### PR TITLE
Render error content in a floating panel

### PR DESCRIPTION
**Pull request by ccschmitz**
## Summary

Renders the content for the error page in a floating panel, similar to what we're doing on sessions.

image

Also updates the UI when an error can't be loaded so it matches what we have in the empty state for sessions. Here's a comparison of the existing and the new callout.

**Old** (yes, it's not centered)
image

**New**
image

## How did you test this change?

Local click test.

## Are there any deployment considerations?

N/A only FE changes.

### CODE REVIEW

File: packages/ui/src/css/vars.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ export const vars = {
 		touchable: 'transform 0.2s cubic-bezier(0.02, 1.505, 0.745, 1.235)',
 	},
 	shadows: {
-		small: '0px 2px 8px -2px rgba(59, 59, 59, 0.08)',
+		small: '0px 6px 12px -2px rgba(59, 59, 59, 0.12)',
```
Comment (ccschmitz) [inline packages/ui/src/css/vars.ts:30]: We weren't using this shadow anywhere and we were importing a shadow from the button's stylesheet a few places. Figured it made more sense to live in the main `shadows` object.

File: frontend/src/pages/ErrorsV2/styles.css.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	padding: '8px',
+})
+
+export const errorDetails = style({
+	background: 'white',
+	borderRadius: '4px',
+	display: 'flex',
+	height: '100%',
+	marginLeft: '0',
+	padding: `var(--size-large) var(--size-xxLarge)`,
```
[the remark this answers is not in the document]
Comment (ccschmitz) [reply frontend/src/pages/ErrorsV2/styles.css.ts:23]: Oh, yes! Thanks for calling these out.

[the remark this answers is not in the document]
Comment (ccschmitz) [reply frontend/src/pages/ErrorsV2/styles.css.ts]: Fixed in [4c53a10](https://github.com/highlight/highlight/pull/3612/commits/4c53a1027866da49f7c51b02ac37589bb91f0337)

**Comment by ccschmitz** [comment]
@aptlin ready for another look whenever you have a moment!

**Review by Vadman97 — APPROVED** [review]
looks awesome! 🚢

**Pull request merged**